.editorconfig

```diff
@@ -0,0 +1,6 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
```

Comment:
Why do we need it? As far as I see [scalafmt should handle that](https://github.com/scalameta/scalafmt/pull/1333). I guess it doesn't work?

The problem with this approach I see, is the fact that we cannot be sure it it will be taken into consideration. But maybe this is not a problem, because, as I mentioned somewhere before, the current mechanism can be skipped by adding `--no-verify` to `git commit` or editing the file though GH UI.

Reply:
scalafmt doesn't touch Markdown or bash files. This is a generic formatting rule for editors that understand EditorConfig (e.g. IDEA)